doc/api_cli.rst

```diff
@@ -57,13 +57,34 @@ When packaging your custom command, we recommend you to define a virtual provide
 
       Depending on other demands and the user's configuration, this might or might not correctly trigger metadata download for the available repositories.
 
+    .. attribute:: load_system_repo
+
+      If ``True`` DNF will load information about installed packages from the local RPMDB into the sack during :meth:`dnf.Base.fill_sack` Defaults to ``True``.
+
+    .. attribute:: cacheonly
+
+      When ``True``, DNF will run entirely from the system cache (equivalent of ``-C`` command line option). Defaults to ``False``.
+
+    .. attribute:: fresh_metadata
+
+      ``False`` means that (even expired) cached repository metadata will be used. When ``True`` the expired repository metadata caches are synchronized with server. Defaults to ``True``.
+
+    .. attribute:: freshest_metadata
+
+      If ``True`` metadata caches for all enabled repositories are forcibly expired before the sack is activated. Defaults to ``False``.
+
+    .. attribute:: changelogs
+
+      If ``True`` also the repository metadata containing changelogs for packages will be downloaded. Defaults to ``False``.
```

Comment:
I think I'd put a comma after _{If,When} `True`_ in all three occurrences above, though I think either is ok (with comma or without). This formulation repeats a lot throughout the documentation, but at least make these four consistent :slightly_smiling_face: